feature/bgp/multihop/README.md

```diff
@@ -0,0 +1,81 @@
+
+## Procedure 
+
+### Configuration
+
+1)  Create the topology below:
+
+```mermaid
+graph LR; 
+A[ATE:Port1] -- EBGP --> B[Port1:DUT:Port2];
+B ----> C[Port2:ATE] --> eBGP peer;
+```
+
+2)  Configure the eBGP peering session: Establish an eBGP peering relationship between ATE1 and Port 1 on the DUT.
+3)  Assign IP addresses: Configure IPv4 and IPv6 addresses on all relevant interfaces on the DUT, ATE1, and ATE2.
+4)  Create a loopback interface: Configure a loopback interface on the DUT and assign it an IP address. This will serve as the DUT's identifier for the BGP session.
+5)  Establish static routing on the DUT: Configure a static route on the DUT that directs traffic destined for the eBGP peer's IP address (connected to ATE2) through ATE2. This ensures the DUT can reach the peer.
+6)  Establish static routing on ATE2: Configure a static route on ATE2 that directs traffic destined for the DUT's loopback IP address towards the DUT. This allows the eBGP peer on ATE1 to reach and identify the DUT via its loopback address.
+
+### Tests
+
+### RT-1.63.1: Establish eBGP session over multihop
+
+1)  Configure the eBGP peering session: Establish an eBGP peering relationship between the DUT's loopback interface and the eBGP peer connected to ATE2. This means the DUT will use its loopback address as its BGP identifier.
+2)  Specify the neighbor address: Configure the DUT to use the eBGP peer's IP address (connected to ATE2) as the neighbor address for this BGP session.
+3)  Enable multihop: Since the eBGP peer is not directly connected to the DUT, enable the multihop option for this BGP neighbor on the DUT. Set the Time-to-Live (TTL) value to 2 to allow the BGP packets to traverse the intermediate link between the DUT and ATE2.
+4)  Validate session establishment: Confirm that the eBGP session is successfully established between the DUT's loopback interface and the eBGP peer connected to ATE2. This can be verified by checking the BGP neighbor status on both the DUT and the peer.
+5)  Verify BGP updates: Ensure that the DUT and the eBGP peer are exchanging BGP updates correctly, indicating that the peering is functional and routing information is being shared.
+
+### RT-1.63.2: Advertise prefixes over multihop eBGP
+
+1)  Advertise prefixes: Configure the eBGP peer to advertise the following IPv4 and IPv6 prefixes to the DUT:
+    *   BGP-V4 = 203.0.200.0/24
+    *   BGP-V6 = 2001:db8:128:200::/64
+2)  Verify prefix reception: Confirm that the DUT successfully receives the advertised prefixes from the eBGP peer.
+3)  Check routing table: Inspect the DUT's routing table to ensure that the received prefixes have been installed correctly.
+4)  Validate next hop: Verify that the next hop associated with the installed prefixes in the DUT's routing table is the IP address of ATE Port 2. This confirms that the DUT knows to forward traffic for these prefixes towards ATE2.
```

Comment:
Need to specify the paths used for this as there are a couple of options.  I recommend  using the afts table as we generally have not required the bgp rib paths (and adding them creates new dependencies on features only needed for test, which is not desirable if there are suitable alternatives)  and there are no BMP paths in the OC model.


Here are some of the relevant leaves in the AFT tree (and AFTs is something we heavily use for many features)

[/network-instances/network-instance/afts/ipv4-unicast/ipv4-entry/state/prefix](https://openconfig.net/projects/models/schemadocs/yangdoc/openconfig-network-instance.html#network-instances-network-instance-afts-ipv4-unicast-ipv4-entry-state-prefix)

[/network-instances/network-instance/afts/ipv4-unicast/ipv4-entry/state/next-hop-group](https://openconfig.net/projects/models/schemadocs/yangdoc/openconfig-network-instance.html#network-instances-network-instance-afts-ipv4-unicast-ipv4-entry-state-next-hop-group)

[/network-instances/network-instance/afts/next-hop-groups/next-hop-group/state/id](https://openconfig.net/projects/models/schemadocs/yangdoc/openconfig-network-instance.html#network-instances-network-instance-afts-next-hop-groups-next-hop-group-state-id)

[/network-instances/network-instance/afts/next-hop-groups/next-hop-group/next-hops/next-hop/state/index](https://openconfig.net/projects/models/schemadocs/yangdoc/openconfig-network-instance.html#network-instances-network-instance-afts-next-hop-groups-next-hop-group-next-hops-next-hop-state-index)

[/network-instances/network-instance/afts/next-hops/next-hop/state/ip-address](https://openconfig.net/projects/models/schemadocs/yangdoc/openconfig-network-instance.html#network-instances-network-instance-afts-next-hops-next-hop-state-ip-address)

[/network-instances/network-instance/afts/next-hops/next-hop](https://openconfig.net/projects/models/schemadocs/yangdoc/openconfig-network-instance.html#network-instances-network-instance-afts-next-hops-next-hop-state-origin-protocol)